src/components/utils/mutations.ts

```diff
@@ -7,29 +7,20 @@
 export function isMutationBelongsToElement(mutationRecord: MutationRecord, element: Element): boolean {
   const { type, target, addedNodes, removedNodes } = mutationRecord;
 
-  /**
-   * In case of removing the whole text in element, mutation type will be 'childList',
-   * 'removedNodes' will contain text node that is not existed anymore, so we can't check it with 'contains' method
-   * But Target will be the element itself, so we can detect it.
-   */
```

Comment:
why comments are removed?

Reply:
@neSpecc Hi, I've restructured the code a little bit and replaced the part this comment was addressed to.

Reply:
So you need to write the new comments explaining all logical parts.

P.S. Please, do not resolve other people's comments. Reply instead.

Reply:
Added some comments